libqfieldsync/offline_converter.py

```diff
@@ -135,7 +139,16 @@ def convert(self, reload_original_project: bool = True) -> None:
         """
         project = QgsProject.instance()
         self.original_filename = Path(project.fileName())
-        self.backup_filename = make_temp_qgis_file(project)
+        if (
+            self.export_type == ExportType.Cable
+            and self._export_filename
+            and self._export_title
+        ):
+            self.backup_filename = make_temp_qgis_file(
+                project, self._export_filename, self._export_title
+            )
+        else:
+            self.backup_filename = make_temp_qgis_file(project)
```

Comment:
```suggestion
        self.backup_filename = make_temp_qgis_file(
            project, self._export_filename, self._export_title
        )
```

This should work the same no matter cable or not.